int64 robot_id
bool ready